### DISCUSSION TITLE
location bias too strict or too relaxed

### DISCUSSION THREAD

**Discussion by karussell** [komoot/photon#814] [Q&A]
Related to https://github.com/komoot/photon/pull/580

For the following query the location bias (in berlin) acts like a filter:

https://photon.komoot.io/api/?q=hoyerswerda&limit=5&lat=52.525621&lon=13.396111

And the city hoyerswerda is not in the list although only 2 items are shown.

If I set zoom=9 it ignores the bias and in this case the results from Berlin (with street "Hoyerswerdaer Straße") even disappear:

https://photon.komoot.io/api/?q=hoyerswerda&limit=5&lat=52.525621&lon=13.396111&zoom=9&limit=100

From experiments it seems that I should use in this case zoom=11 to still include one result for Berlin, but I want them to come first (IMO most logical) and I can increase the zoom to achieve this, but then the location bias seems to act a filter and reduces the result count, which is unexpected. Maybe the exponential function should get a small offset so that it never can be 0?

**Comment by lonvia** [comment]
Try increasing the 'location_bias_scale' to something like 0.3. The tricky part here is finding the right balance between well known places and those near the location bias.
